### PR TITLE
Add TWiRx

### PR DESCRIPTION
**Pull request by Ticki**
I fucked up by sending a PR to the generated repo, so here's the new patch.

### CODE REVIEW

File: static/css/redox.css [hunk 1]
```diff
@@ -2,18 +2,23 @@
   box-sizing: border-box;
 }
 
-head, body {
-  font-family: 'Lato', 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+a {
+  font: Helvetica, arial, nimbussansl, liberationsans,freesans, clean, sans-serif
```
Comment (hauleth) [inline static/css/redox.css]: Why so? I mean, why not Lato?

File: static/css/redox.css [hunk 2]
```diff
@@ -43,11 +48,11 @@ nav li a {
 }
 
 .topbar {
-  width: 100%;
+  width: 92%;
   position: fixed;
-  top: 0;
-  left: 0;
-  height: 4em;
+  top: 0px;
```
Comment (hauleth) [inline static/css/redox.css]: `px` is not needed and it is common practice to omit unit when using `0` as a value. I would like to keep that.

File: static/css/redox.css (hunk 1 above)
Comment (Ticki) [inline static/css/redox.css]: Because the other look more modern. I can change it if you want.

File: static/css/redox.css (hunk 2 above)
Comment (Ticki) [inline static/css/redox.css]: My bad.

File: static/css/redox.css [hunk 3]
```diff
@@ -43,11 +48,11 @@ nav li a {
...
+  left: 4%;
```
Comment (hauleth) [inline static/css/redox.css]: Why not using `padding`/`margin` but `left`? Any reasoning behind that?

File: static/css/redox.css (hunk 2 above)
Comment (Ticki) [inline static/css/redox.css]: Fixed.

File: static/css/redox.css [hunk 4]
```diff
@@ -43,11 +48,11 @@ nav li a {
...
+  height: 4.8em;
```
Comment (hauleth) [inline static/css/redox.css]: Remember that I set `border-box` for all elements.

File: static/css/redox.css [hunk 5]
```diff
@@ -2,18 +2,23 @@
   box-sizing: border-box;
 }
 
-head, body {
-  font-family: 'Lato', 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+a {
+  font: Helvetica, arial, nimbussansl, liberationsans,freesans, clean, sans-serif
+  font-weight: bold;
+  box-sizing: border-box;
```
Comment (hauleth) [inline static/css/redox.css]: Not needed as I set `border-box` for everything.

File: static/css/redox.css [hunk 6]
```diff
@@ -2,18 +2,23 @@
...
+  color: #4078C0;
+  word-wrap: break-word;
```
Comment (hauleth) [inline static/css/redox.css]: :+1:

File: static/css/redox.css [hunk 7]
```diff
@@ -2,18 +2,23 @@
...
 }
 
 body {
-  margin-top: 5em;
+  font-family: "Helvetica Neue", Helvetica, "Segoe UI",Arial, freesans, sans-serif;
```
Comment (hauleth) [inline static/css/redox.css]: Why there is different font choice for `body` and `a`?

Comment (Ticki) [inline static/css/redox.css]: Will fix asap.

Comment (Ticki) [inline static/css/redox.css]: Fixed

File: static/css/redox.css (hunk 1 above)
Comment (hauleth) [inline static/css/redox.css]: IMHO Lato looks better and for sure Lato has better license and is available on all platforms.

Also it is 3rd most popular font according to Google Web Fonts.

BTW I lato (summer in polish and I'm Pole).

**Comment by Ticki** [comment]
If you don't like the style change, I can remove it. Sorry for being such a jerk and working on the master branch...

**Pull request merged**